CHANGELOG_PENDING.md

```diff
@@ -19,6 +22,8 @@
 
 ### IMPROVEMENTS:
 
-- [CircleCI] \#3497 Move release management to CircleCI
+- [circle] \#3497 Move release management to CircleCI
```

Comment:
This is not clear in the release process documentation. Specifically, how do we name changes in the top-level / root package directory and in other non-golang directories, e.g. `DOCKER`, `.circleci`, etc.
ref: https://github.com/tendermint/tendermint/pull/3503